main.py

```diff
@@ -1,7 +1,47 @@
 import init_django_orm  # noqa: F401
-
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    genres = [
+        ("Western",),
+        ("Action",),
+        ("Drama",),
+    ]
+
+    actors = [
+        ("George", "Clooney"),
+        ("Keanu", "Reeves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson"),
+    ]
+
+    for (name,) in genres:
+        Genre.objects.create(name=name)
+
+    for first_name, last_name in actors:
+        Actor.objects.create(first_name=first_name, last_name=last_name)
+
+    Genre.objects.filter(name="Drama").update(name="Drama")
+    Actor.objects.filter(
+        first_name="George",
+        last_name="Clooney"
+    ).update(
+        last_name="Clooney"
+    )
+
+    Actor.objects.filter(
+        first_name="Kianu",
+        last_name="Reaves"
+    ).update(
+        first_name="Keanu",
+        last_name="Reeves"
+    )
```

Comment:
It seems like you're trying to update the first name and last name of an actor. However, the names you're searching for ('Kianu', 'Reaves') don't match the names you initially added ('Keanu', 'Reeves'). This means that the update operation won't find a match and won't do anything. Please check the names you're using.